src/component/footer/footer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styles from "./footer.module.scss";
+
+const Footer = ({
+  handleClickBack,
+  handleClick,
+  previousPageLink,
+  nextPageLink,
+}) => {
+  return (
+    <footer className={styles.footer}>
+      <div className={styles.footer__backoptions} onClick={handleClickBack}>
```

Comment:
It is better to use styles["footer__backoptions"] for all the scss module name
Please change all other class name as well.